README.md

```diff
@@ -95,39 +95,32 @@ Confirm your PHP installation has curl enabled
 
 This quick example assumes that you have a local riak cluster running on port 8098
```

Comment:
Can you add to this intro for the example that it assumes we are using composer to pull your dependencies into the vendor directory? If not, I can do it post merge.